workdir/copy/link/run.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/sh
+
+plat=""
+arch=""
+fs=""
+net=0
+kvm=0
+extra_boot_args=""
+
+prepare_initrd()
+{
+    cd fs0
+    find -depth -print | tac | bsdcpio -o --format newc > ../fs0.cpio
+    cd ..
+}
+
+
+prepare_qemu_net()
+{
+    sudo ip link set dev virbr0 down 2> /dev/null
+    sudo ip link del dev virbr0 2> /dev/null
+    sudo ip link add dev virbr0 type bridge
+    sudo ip address add 172.44.0.1/24 dev virbr0
+    sudo ip link set dev virbr0 up
+}
+
+prepare_fc_net()
+{
+    sudo ip link set dev tap0 down 2> /dev/null
+    sudo ip link del dev tap0 2> /dev/null
+    sudo ip tuntap add dev tap0 mode tap
+    sudo ip address add 172.45.0.1/24 dev tap0
+    sudo ip link set dev tap0 up
+}
+
+run_qemu()
+{
+    if test "$arch" = "aarch64"; then
+        cmd="qemu-system-aarch64 -kernel $kernel_image -nographic -machine virt -cpu max"
+    elif test "$arch" = "x86_64"; then
+        cmd="qemu-system-x86_64 -kernel $kernel_image -nographic"
+        if test "$kvm" -eq 1; then
+            cmd="$cmd -enable-kvm -cpu host"
+        fi
+    fi
+    if test "$net" -eq 1; then
+        prepare_qemu_net
+        cmd="$cmd -netdev bridge,id=en0,br=virbr0 -device virtio-net-pci,netdev=en0 -append \"netdev.ipv4_addr=172.44.0.2 netdev.ipv4_gw_addr=172.44.0.1 netdev.ipv4_subnet_mask=255.255.255.0 -- $extra_boot_args\""
+    fi
+    if test "$fs" = "initrd"; then
+	prepare_initrd
+        cmd="$cmd -initrd fs0.cpio"
+    fi
+    if test "$fs" = "9pfs"; then
+        cmd="$cmd -fsdev local,id=myid,path=$(pwd)/fs0,security_model=none -device virtio-9p-pci,fsdev=myid,mount_tag=fs0,disable-modern=on,disable-legacy=off"
+    fi
+
+    echo "Running command:"
+    echo "$cmd"
+    eval "$cmd"
+}
+
+prepare_fc_json()
+{
+    cp fc_template.json fc.json
```

Comment:
I think you forgot to add the `fc_template.json` to the commit?

Reply:
Yes, you're right, I will add it.